e2etest/pkg/frr/bgp.go

```diff
@@ -124,3 +124,27 @@ func ContainsCommunity(exec executor.Executor, community string) error {
 	}
 	return nil
 }
+
+func allroutesMatchLocalPref(routes map[string]bgpfrr.Route, localPref uint32) error {
```

Comment:
nit: allRoutes

Reply:
Also, if this is not exported, makes sense to have it below `RoutesMatchLocalPref`